src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -1605,17 +1605,12 @@ private void Open(object parameters)
                 }
 
                 var directoryName = Path.GetDirectoryName(filePath);
-                //Checks if the file that is being opened is in the trusted list.
-                bool bShowFileTrustWarning = false;
-                if(!PreferenceSettings.TrustedLocations.Contains(directoryName) && DynamoModel.IsTestMode == false)
-                {
-                    HomeSpace.RunSettings.ForceAutomaticWithoutRun = true;
-                    bShowFileTrustWarning = true;
-                }
                 ExecuteCommand(new DynamoModel.OpenFileCommand(filePath, forceManualMode));
-
-                if (FileTrustViewModel != null && bShowFileTrustWarning == true)
+                if (!PreferenceSettings.TrustedLocations.Contains(directoryName) 
+                    && DynamoModel.IsTestMode == false
+                    && FileTrustViewModel != null)
                 {
+                    HomeSpace.RunSettings.ForceAutomaticWithoutRun = true;
```

Comment:
@RobertGlobant20 I included a minimal fix for now and plan to rethink about this boolean with you a bit to cover the periodic run mode as well